src/main/io/serial_msp.c

```diff
@@ -484,7 +484,7 @@ void mspInit(serialConfig_t *serialConfig)
         activeBoxIds[activeBoxIdCount++] = BOXBARO;
     }
 
-    if (sensors(SENSOR_ACC) || sensors(SENSOR_MAG)) {
+    if (sensors(SENSOR_ACC) && sensors(SENSOR_MAG)) {
```

Comment:
@TheAngularity , I think `||` is correct, why do you propose changing it to `&&`?